unified_range/utils.py

```diff
@@ -88,23 +92,21 @@ def transform_to_semver(unified_spec: str) -> str:
                 if restriction.has_inclusive_upper:
                     lt_lte = operators["lte"]
                 semvers.append(
-                    "{}{} {}{}".format(gt_gte, restriction.lower_bound,
-                                       lt_lte, restriction.upper_bound))
+                    f"{gt_gte}{restriction.lower_bound}{separator}"
```

Comment:
better to change from f strings to named format args

Reply:
I opted for the non-named option, since it was too verbose imo.